Styling

estebanmual opened the pull request:
In this milestone I: 
  - Matched app's design with Zeplin's design

estebanmual committed:
Apply styling to the project following the Zeplin's design

elmar8287 commented on src/components/AddBook.jsx:
```diff
@@ -1,19 +1,21 @@
+    <div className={style.AddBook}>
+      <h2 className={style.AddBook__title}>ADD NEW BOOK</h2>
+      <form className={style.AddBook__form}>
+        <input className={`${style.AddBook__input} ${style.AddBook__inputTitle}`} type="text" value={title || ''} placeholder="Book title" onChange={(e) => setTitle(e.target.value)} required />
+        <input className={`${style.AddBook__input} ${style.AddBook__inputAuthor}`} type="text" value={author || ''} placeholder="Author" onChange={(e) => setAuthor(e.target.value)} required />
```
- [x] Please, consider adding the "Category" choosing possibility, as mentioned in the original design.
- [x] Additionally, please, consider to not forget to place the category name on top of the title, as also mentioned on the screen below

> Original 
<img width="705" alt="Screen Shot 2022-04-01 at 00 17 10" src="https://user-images.githubusercontent.com/49064106/161142866-4cfb6f4a-8ff5-4997-8133-a41687448504.png">

elmar8287 reviewed (changes requested):
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- Perfect styling 👍 
- No linter issue 💯 

## Required Changes ♻️

- [x]  _Check the comments under the review._

## Optional suggestions

- [x] Please, fix the broken links for demo live and MIT.md file 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

estebanmual committed:
Update README.md

estebanmual committed:
Update README.md

estebanmual committed:
Create MIT.md

estebanmual committed:
Added category field to form and save it in the API
 On branch feature/styling
	modified:   src/components/AddBook.jsx
	modified:   src/redux/books/books.js
	modified:   src/styling/AddBook.module.css

EricMbouwe reviewed (approved):
### Hi @estebanmual, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
